File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.js
function solution(num_list) {
    var answer = 0
    var oddsum =0;
    var evensum=0;
    for(var i=0 ; i<num_list.length ;i++){
        if(i%2==0) oddsum+=num_list[i];
        else evensum+=num_list[i];
    }
    if(oddsum>evensum) answer=oddsum;
    else answer=evensum;
    return answer;
}

File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.py
def solution(num_list):
    answer = 0
    oddsum = 0;
    evensum = 0;
    for i in range(len(num_list)):
        if i%2==0: oddsum+=num_list[i]
        else: evensum+=num_list[i]
        if oddsum>evensum: answer=oddsum
        else: answer=evensum
    return answer